.github/workflows/release-maven-image.yaml

```diff
@@ -0,0 +1,68 @@
+name: release-maven-image
+
+on:
+  workflow_dispatch:
+    inputs:
+      releaseVersion:
+        description: "Default version to use when preparing a release."
+        required: true
+        default: "X.Y.Z"
+      developmentVersion:
+        description: "Default version to use for new local working copy."
+        required: true
+        default: "X.Y.Z-SNAPSHOT"
+      app-path:
+        type: choice
+        description: Service-Name
+        required: true
+        options:
+          - refarch-backend
+          - refarch-eai
+
+jobs:
+  release-maven:
+    runs-on: ubuntu-latest
+    outputs:
+      MVN_ARTIFACT_ID: ${{ steps.maven-release-step.outputs.MVN_ARTIFACT_ID }}
+      ARTIFACT_NAME: ${{ steps.maven-release-step.outputs.artifact-name }}
+    env:
+      TZ: Europe/Berlin # timezone
+    steps:
+      - id: maven-release-step
+        uses: it-at-m/.github/.github/actions/action-maven-release@main
+        with:
+          app-path: "/${{ github.event.inputs.app-path }}"
+          releaseVersion: "${{ github.event.inputs.releaseVersion }}"
+          developmentVersion: "${{ github.event.inputs.developmentVersion }}"
+          java-version: 17
```

Comment:
```suggestion
          java-version: 21
```

Reply:
oder zeile einfach komplett raus werfen. und nur in den templates das default angeben